app/lib/methods/helpers/showToast.ts

```diff
@@ -0,0 +1,4 @@
+import { LISTENER } from '../../../containers/Toast';
+import EventEmitter from '../../../utils/events';
+
+export const showToast = (message: string): void => EventEmitter.emit(LISTENER, { message });
```

Comment:
I created this helper method because this same logic is replicated in several files, but the correct one should be a function with this centralized logic, even to facilitate future maintenance.

About the name, that's what I thought was best hahaha, I'm open to opinions.